### ISSUE TITLE
[Suggestion] x-ray logic elements

### ISSUE THREAD

**Issue by AlienXtream** [tsunamayo/Starship-EVO#2166]
### Short Description of the issue:
having logic components (anything that can be wired) show through walls while holding the wire tool. also having a mode to show all connections in range rather than just the active item
### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
Yes that is something I want to work on. Showing all connections might be a bit too much, I might show only link that are in the currently selected chain though. Or I might add a toggle, I will test. Thanks

**Comment by AlienXtream** [comment]
i was indeed meaning a toggle. not even that. holding the "reload" could show up a dimmer variant of the wires so you can see at a glance what's connected where. yes it would be overwhelming but it would be good for advanced users. think of it like a standard calculator and a scientific calculator where the standard is good for simple stuff but more advanced things need more advanced tools to do effectively.
